base/shared_buffer_manager.cpp

```diff
@@ -9,13 +9,13 @@ SharedBufferManager & SharedBufferManager::instance()
 
 void SharedBufferManager::clearReserved()
 {
-  threads::MutexGuard g(m_mutex);
+  std::unique_lock<std::mutex> g(m_mutex);
```

Comment:
std::lock_guard is better and more informative here and below. We don't share or transfer lock here, right?